dask_cloudprovider/aws/ecs.py

```diff
@@ -179,6 +179,7 @@ async def _update_task(self):
                             cluster=self.cluster_arn, tasks=[self.task_arn]
                         )
                     )["tasks"]
+                    await asyncio.sleep(5)
```

Comment:
This just seems like a flat out oversight -- they've already got the `wait_duration` variable, but they're only using it in the case of the `ThrottlingException`.

If you changed this line you're adding to `asynio.sleep(wait_duration)` I bet that would solve the issue too -- I bet we'd make 100x less calls to the API. And I'd think we could put up a PR for it in their repo and have it accepted.